src/app/public/modules/modal/types/modal-before-close-handler.ts

```diff
@@ -0,0 +1,3 @@
+export class SkyModalBeforeCloseHandler {
+  constructor(public close: Function) {}
```

Comment:
The name of the "close" method should be something similar to `triggerClose` or `closeModal`, since we're not "closing" the handler.

Reply:
done